openlibrary/templates/type/edition/view.html

```diff
@@ -412,7 +412,7 @@ <h3 class="collapse">$_("Contributors")</h3>
 
           $if has_any("physical_format", "pagination", "physical_dimensions", "weight"):
             <div class="section">
-                <h3 class="list-header collapse">$_("The Physical Object")</h3>
+                <h3 class="edition-header">$_("The Physical Object")</h3>
```

Comment:
These should still be list-header.

The only issue is with the generic h3's that don't match list-header.